src/test/resources/feature/WriteReview.feature

```diff
@@ -0,0 +1,17 @@
+@Irina
+Feature:   Write Review
+
+  Scenario: WriteReview - Positive
+    Given user is logged in
+    And "Home" is displayed
+    And user clicks on "item1"
+    And "item1's detailed page" is displayed
```

Comment:
Same here, we can assert that this page is displayed in previous step

Reply:
Done